src/MigrationsGenerator/DBAL/Support/FilterTables.php

```diff
@@ -40,7 +40,7 @@ protected function isPgSQLWantedTable(string $table): bool
         }
 
         // Schema name defined in Laravel framework.
-        $schema = DB::connection()->getConfig('schema');
+        $schema = $this->setting->getConnection()->getConfig('schema');
```

Comment:
```suggestion
        $schema = app(MigrationsGeneratorSetting::class)->getConnection()->getConfig('schema');
```

Reply:
Thank you @krewenki  for the fix! Please use `app` resolver as caller of this trait may not have `$this-setting` initiated.